config/docker/php/docker-entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+COMMAND="composer install --prefer-dist --no-progress --no-suggest --no-interaction"
+
+if [[ "prod" = "${APP_ENV}" ]]; then
+    COMMAND="${COMMAND} --no-dev  --optimize-autoloader --classmap-authoritative"
+fi
+
+if [[ ! -z "${HOST_UID}" ]]; then
+    deluser www-data
```

Comment:
No need to delete user. You can modify by it `/usr/sbin/usermod -u "$HOST_UID" www-data`